WBooks/LandingScreen/LandingScreenView.swift

```diff
@@ -14,6 +14,15 @@ class LandingScreenView: UIView, NibLoadable {
             loginButton.layer.cornerRadius = 25
             loginButton.layer.borderWidth = 2
             loginButton.layer.borderColor = UIColor.white.cgColor
+            loginButton.setTitle("LOGIN_BUTTON".localized(), for: UIControl.State.normal)
```

Comment:
we use to avoid the "UIControl.State" in order to right less 😜same for the "UIColor"